moon/bulk_test.sh

```diff
@@ -33,6 +35,7 @@ while getopts "r:t:s:f:l:hw:ud:p:" opt; do
             echo " -u .. only record unsuccessful run logs"
             echo " -d <log dir prefix>"
             echo " -p <times to repeat each seed>"
+            echo " -v <launch VSLAM docker>"
```

Comment:
as discussed over e-mail the result will be ONE docker, so we will have to integrate VSLAM into single docker

Reply:
Keeping it separate for now for development purposes.